docs/source/upgrade-guide/index.md

```diff
@@ -253,31 +253,62 @@ It is highly recommended that you use latest Node 16.
 2. Change your root project `Makefile` to include these commands:
 
 ```diff
++.PHONY: install
++install: ## Install the frontend
++       @echo "Install frontend"
++       $(MAKE) omelette
++       $(MAKE) preinstall
++       $(MAKE) develop
```

Comment:
Is it possible that a developer would want to run `make omelette`, `make preinstall`, or `make develop` directly? If so, then there should be help text for each target below.

Reply:
ups... I've found a bug in the middle. Thanks to point me to double check this. Commit incoming!